packages/realm/src/Configuration.ts

```diff
@@ -231,7 +249,7 @@ export function validateConfiguration(config: unknown): asserts config is Config
  * Validate the data types of the fields of a user-provided realm schema.
  */
 export function validateRealmSchema(realmSchema: unknown): asserts realmSchema is Configuration["schema"][] {
-  assert.array(realmSchema, "the realm schema");
+  assert.array(realmSchema, "schema (the realm schema)");
```

Comment:
I'd say this would suffice - we want to keep these as short as possible to avoid information overload on the developers.

```suggestion
  assert.array(realmSchema, "realm schema");
```

In other parts of the code-base, I've actually tried to only use property names for this - to keep it even simpler, but perhaps that's too extreme.

Reply:
Thanks I'll update this!